pkgs/tools/misc/screamingfrogseospider/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, lib, fetchurl, dpkg, oraclejre }:
```

Comment:
Just out of curiosity, other JREs don't work?

Reply:
At the time, only oraclejre and jetbrains jres had support for OpenJFX and the jetbrains jre straight up didn't work. IIRC at the time this was still in flux, so will review again given that it's been a few months.